Point_set_processing_3/include/CGAL/Point_with_normal_3.h

```diff
@@ -138,17 +138,16 @@ struct Normal_of_point_with_normal_pmap
 
   typedef Point_with_normal key_type;
   typedef Vector value_type;
-  typedef value_type& reference;
+  typedef const value_type& reference;
   typedef boost::lvalue_property_map_tag category;
 
   /// Access a property map element
-  reference operator[](key_type& pwn) const { return pwn.normal(); }
+  value_type& operator[](key_type& pwn) const { return pwn.normal(); }
 
```

Comment:
If `reference` is `const value_type&`, then the lvalue property map is non-mutable. In that case, `pmap[key]` should return a `const value_type&`.

http://www.boost.org/doc/libs/1_61_0/libs/property_map/doc/LvaluePropertyMap.html